<launch>

    <arg name="project" default="feature_matching"/>

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find lio_sam_custom)/launch/include/config/rviz.rviz" />

</launch>
